### ISSUE TITLE
Get-AzRecoveryServicesBackupProtectionPolicy - improper time display

### ISSUE THREAD

**Issue by peppekerstens** [Azure/azure-powershell#10546]
## Description
When using Get-AzRecoveryServicesBackupProtectionPolicy to display the existing policies, it displays wrong time.

I am aware of the fact that time displayed is universal time. The issue is that two different policies, created via the Azure Portal GUI with **same** start time and timezone show a **different** time when queried by Get-AzRecoveryServicesBackupProtectionPolicy.

## Steps to reproduce
Probably hard to reproduce on demand. However, I do have 2 subscriptions which have this issue.
Code used to reguest the exisiting policies: 

```powershell
get-azsubscription | where name -like "*XXXXX*" | select-azsubscription
$rsVault = Get-AzRecoveryServicesVault -Name 'XXXXX'
Get-AzRecoveryServicesBackupProtectionPolicy -VaultId $rsVault.ID | Where Name -Like "*XXXX*"
```

## Environment data

```powershell
$psversiontable

Name                           Value
----                           -----
PSVersion                      5.1.18956.1000
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.18956.1000
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

## Module versions

```powershell
get-package az

Name                           Version          Source                           ProviderName
----                           -------          ------                           ------------
Az                             3.0.0            https://www.powershellgallery... PowerShellGet
```

## Debug output
Be ware: both policies show 08:00pm (+1:00 Amsterdam) in the Azure GUI...

```powershell
Get-AzRecoveryServicesBackupProtectionPolicy -VaultId $rsVault.ID | Where Name -Like "*Sql*"
DEBUG: 7:17:24 PM - GetAzureRmRecoveryServicesBackupProtectionPolicy begin processing with ParameterSet 'NoParamSet'.
DEBUG: 7:17:24 PM - using account id 'xxxxxxxx'...
DEBUG: Inside Restore. Going to create ResourceClient.
DEBUG: [Common.Authentication]: Authenticating using Account: 'xxxxxxxx', environment: 'AzureCloud', tenant: '048e4758-eb3c-4d3e-8534-630fbf0bacd5
'
DEBUG: [Common.Authentication]: Authenticating using configuration values: Domain: '048e4758-eb3c-4d3e-8534-630fbf0bacd5', Endpoint: 'https://login.micr
osoftonline.com/', ClientId: '1950a258-227b-4e31-a9cf-717495945fc2', ClientRedirect: 'urn:ietf:wg:oauth:2.0:oob', ResourceClientUri: 'https://management
.core.windows.net/', ValidateAuthority: 'True'
DEBUG: [Common.Authentication]: Acquiring token using context with Authority 'https://login.microsoftonline.com/048e4758-eb3c-4d3e-8534-630fbf0bacd5/', 
CorrelationId: '00000000-0000-0000-0000-000000000000', ValidateAuthority: 'True'
DEBUG: [Common.Authentication]: Acquiring token using AdalConfiguration with Domain: '048e4758-eb3c-4d3e-8534-630fbf0bacd5', AdEndpoint: 'https://login.
microsoftonline.com/', ClientId: '1950a258-227b-4e31-a9cf-717495945fc2', ClientRedirectUri: urn:ietf:wg:oauth:2.0:oob
DEBUG: [ADAL]: Information: 2019-11-14T18:17:24.1854473Z: 6f1dc99d-5e39-49a2-a43d-0c0c9b696a57 - LoggerBase.cs: ADAL PCL.Desktop with assembly version '
3.19.2.6005', file version '3.19.50302.0130' and informational version '2a8bec6c4c76d0c1ef819b55bdc3cda2d2605056' is running...

DEBUG: [ADAL]: Information: 2019-11-14T18:17:24.1854473Z: 6f1dc99d-5e39-49a2-a43d-0c0c9b696a57 - LoggerBase.cs: ADAL PCL.Desktop with assembly version '
3.19.2.6005', file version '3.19.50302.0130' and informational version '2a8bec6c4c76d0c1ef819b55bdc3cda2d2605056' is running...

DEBUG: [ADAL]: Information: 2019-11-14T18:17:24.1854473Z: 6f1dc99d-5e39-49a2-a43d-0c0c9b696a57 - LoggerBase.cs: === Token Acquisition started: 
	CacheType: null
	Authentication Target: User
	, Authority Host: login.microsoftonline.com

DEBUG: [ADAL]: Information: 2019-11-14T18:17:24.1854473Z: 6f1dc99d-5e39-49a2-a43d-0c0c9b696a57 - LoggerBase.cs: === Token Acquisition started:
	Authority: https://login.microsoftonline.com/048e4758-eb3c-4d3e-8534-630fbf0bacd5/
	Resource: https://management.core.windows.net/
	ClientId: 1950a258-227b-4e31-a9cf-717495945fc2
	CacheType: null
	Authentication Target: User
	

DEBUG: [ADAL]: Verbose: 2019-11-14T18:17:24.1854473Z: 6f1dc99d-5e39-49a2-a43d-0c0c9b696a57 - LoggerBase.cs: Loading from cache.

DEBUG: [ADAL]: Verbose: 2019-11-14T18:17:24.1854473Z: 6f1dc99d-5e39-49a2-a43d-0c0c9b696a57 - LoggerBase.cs: Loading from cache.

DEBUG: [ADAL]: Information: 2019-11-14T18:17:24.1864454Z: 00000000-0000-0000-0000-000000000000 - LoggerBase.cs: Deserialized 9 items to token cache.

DEBUG: [ADAL]: Verbose: 2019-11-14T18:17:24.1864454Z: 6f1dc99d-5e39-49a2-a43d-0c0c9b696a57 - LoggerBase.cs: Looking up cache for a token...

DEBUG: [ADAL]: Verbose: 2019-11-14T18:17:24.1864454Z: 6f1dc99d-5e39-49a2-a43d-0c0c9b696a57 - LoggerBase.cs: Looking up cache for a token...

DEBUG: [ADAL]: Information: 2019-11-14T18:17:24.1864454Z: 6f1dc99d-5e39-49a2-a43d-0c0c9b696a57 - LoggerBase.cs: An item matching the requested resource 
was found in the cache

DEBUG: [ADAL]: Information: 2019-11-14T18:17:24.1864454Z: 6f1dc99d-5e39-49a2-a43d-0c0c9b696a57 - LoggerBase.cs: An item matching the requested resource 
was found in the cache

DEBUG: [ADAL]: Information: 2019-11-14T18:17:24.1864454Z: 6f1dc99d-5e39-49a2-a43d-0c0c9b696a57 - LoggerBase.cs: 56.6971425766667 minutes left until toke
n in cache expires

DEBUG: [ADAL]: Information: 2019-11-14T18:17:24.1864454Z: 6f1dc99d-5e39-49a2-a43d-0c0c9b696a57 - LoggerBase.cs: 56.6971425766667 minutes left until toke
n in cache expires

DEBUG: [ADAL]: Information: 2019-11-14T18:17:24.1864454Z: 6f1dc99d-5e39-49a2-a43d-0c0c9b696a57 - LoggerBase.cs: A matching item (access token or refresh
 token or both) was found in the cache

DEBUG: [ADAL]: Information: 2019-11-14T18:17:24.1864454Z: 6f1dc99d-5e39-49a2-a43d-0c0c9b696a57 - LoggerBase.cs: A matching item (access token or refresh
 token or both) was found in the cache

DEBUG: [ADAL]: Information: 2019-11-14T18:17:24.1864454Z: 6f1dc99d-5e39-49a2-a43d-0c0c9b696a57 - LoggerBase.cs: === Token Acquisition finished successfu
lly. An access token was returned: Expiration Time: 11/14/2019 7:14:06 PM +00:00

DEBUG: [ADAL]: Information: 2019-11-14T18:17:24.1864454Z: 6f1dc99d-5e39-49a2-a43d-0c0c9b696a57 - LoggerBase.cs: === Token Acquisition finished successfu
lly. An access token was returned: Expiration Time: 11/14/2019 7:14:06 PM +00:00Access Token Hash: O9BMcvFsZHaXVyHP7i1u/aGPxI0I1D6HY3N3GuXnxbw=
	 User id: 06e0ff4c-6ec9-482e-b436-927c3a297f3f

DEBUG: Client Created successfully
DEBUG: Input params - Name:NULL, WorkloadType: NULL, BackupManagementType:NULL, ParameterSetName: NoParamSet
DEBUG: going to query service to get list of policies
DEBUG: [Common.Authentication]: Authenticating using Account: 'xxxxxxx', environment: 'AzureCloud', tenant: '048e4758-eb3c-4d3e-8534-630fbf0bacd5
'
DEBUG: [Common.Authentication]: Authenticating using configuration values: Domain: '048e4758-eb3c-4d3e-8534-630fbf0bacd5', Endpoint: 'https://login.micr
osoftonline.com/', ClientId: '1950a258-227b-4e31-a9cf-717495945fc2', ClientRedirect: 'urn:ietf:wg:oauth:2.0:oob', ResourceClientUri: 'https://management
.core.windows.net/', ValidateAuthority: 'True'
DEBUG: [Common.Authentication]: Acquiring token using context with Authority 'https://login.microsoftonline.com/048e4758-eb3c-4d3e-8534-630fbf0bacd5/', 
CorrelationId: '00000000-0000-0000-0000-000000000000', ValidateAuthority: 'True'
DEBUG: [Common.Authentication]: Acquiring token using AdalConfiguration with Domain: '048e4758-eb3c-4d3e-8534-630fbf0bacd5', AdEndpoint: 'https://login.
microsoftonline.com/', ClientId: '1950a258-227b-4e31-a9cf-717495945fc2', ClientRedirectUri: urn:ietf:wg:oauth:2.0:oob
DEBUG: [ADAL]: Information: 2019-11-14T18:17:24.2579672Z: 41625229-cf17-4786-a848-2a00020cfe17 - LoggerBase.cs: ADAL PCL.Desktop with assembly version '
3.19.2.6005', file version '3.19.50302.0130' and informational version '2a8bec6c4c76d0c1ef819b55bdc3cda2d2605056' is running...

DEBUG: [ADAL]: Information: 2019-11-14T18:17:24.2579672Z: 41625229-cf17-4786-a848-2a00020cfe17 - LoggerBase.cs: ADAL PCL.Desktop with assembly version '
3.19.2.6005', file version '3.19.50302.0130' and informational version '2a8bec6c4c76d0c1ef819b55bdc3cda2d2605056' is running...

DEBUG: [ADAL]: Information: 2019-11-14T18:17:24.2579672Z: 41625229-cf17-4786-a848-2a00020cfe17 - LoggerBase.cs: === Token Acquisition started: 
	CacheType: null
	Authentication Target: User
	, Authority Host: login.microsoftonline.com

DEBUG: [ADAL]: Information: 2019-11-14T18:17:24.2579672Z: 41625229-cf17-4786-a848-2a00020cfe17 - LoggerBase.cs: === Token Acquisition started:
	Authority: https://login.microsoftonline.com/048e4758-eb3c-4d3e-8534-630fbf0bacd5/
	Resource: https://management.core.windows.net/
	ClientId: 1950a258-227b-4e31-a9cf-717495945fc2
	CacheType: null
	Authentication Target: User
	

DEBUG: [ADAL]: Verbose: 2019-11-14T18:17:24.2579672Z: 41625229-cf17-4786-a848-2a00020cfe17 - LoggerBase.cs: Loading from cache.

DEBUG: [ADAL]: Verbose: 2019-11-14T18:17:24.2579672Z: 41625229-cf17-4786-a848-2a00020cfe17 - LoggerBase.cs: Loading from cache.

DEBUG: [ADAL]: Information: 2019-11-14T18:17:24.2579672Z: 00000000-0000-0000-0000-000000000000 - LoggerBase.cs: Deserialized 9 items to token cache.

DEBUG: [ADAL]: Verbose: 2019-11-14T18:17:24.2579672Z: 41625229-cf17-4786-a848-2a00020cfe17 - LoggerBase.cs: Looking up cache for a token...

DEBUG: [ADAL]: Verbose: 2019-11-14T18:17:24.2579672Z: 41625229-cf17-4786-a848-2a00020cfe17 - LoggerBase.cs: Looking up cache for a token...

DEBUG: [ADAL]: Information: 2019-11-14T18:17:24.2579672Z: 41625229-cf17-4786-a848-2a00020cfe17 - LoggerBase.cs: An item matching the requested resource 
was found in the cache

DEBUG: [ADAL]: Information: 2019-11-14T18:17:24.2579672Z: 41625229-cf17-4786-a848-2a00020cfe17 - LoggerBase.cs: An item matching the requested resource 
was found in the cache

DEBUG: [ADAL]: Information: 2019-11-14T18:17:24.2579672Z: 41625229-cf17-4786-a848-2a00020cfe17 - LoggerBase.cs: 56.6959505466667 minutes left until toke
n in cache expires

DEBUG: [ADAL]: Information: 2019-11-14T18:17:24.2579672Z: 41625229-cf17-4786-a848-2a00020cfe17 - LoggerBase.cs: 56.6959505466667 minutes left until toke
n in cache expires

DEBUG: [ADAL]: Information: 2019-11-14T18:17:24.2579672Z: 41625229-cf17-4786-a848-2a00020cfe17 - LoggerBase.cs: A matching item (access token or refresh
 token or both) was found in the cache

DEBUG: [ADAL]: Information: 2019-11-14T18:17:24.2579672Z: 41625229-cf17-4786-a848-2a00020cfe17 - LoggerBase.cs: A matching item (access token or refresh
 token or both) was found in the cache

DEBUG: [ADAL]: Information: 2019-11-14T18:17:24.2579672Z: 41625229-cf17-4786-a848-2a00020cfe17 - LoggerBase.cs: === Token Acquisition finished successfu
lly. An access token was returned: Expiration Time: 11/14/2019 7:14:06 PM +00:00

DEBUG: [ADAL]: Information: 2019-11-14T18:17:24.2579672Z: 41625229-cf17-4786-a848-2a00020cfe17 - LoggerBase.cs: === Token Acquisition finished successfu
lly. An access token was returned: Expiration Time: 11/14/2019 7:14:06 PM +00:00Access Token Hash: O9BMcvFsZHaXVyHP7i1u/aGPxI0I1D6HY3N3GuXnxbw=
	 User id: 06e0ff4c-6ec9-482e-b436-927c3a297f3f

DEBUG: [Common.Authentication]: Renewing Token with Type: 'Bearer', Expiry: '11/14/2019 19:14:06 +00:00', MultipleResource? 'True', Tenant: '048e4758-eb
3c-4d3e-8534-630fbf0bacd5', UserId: 'xxxxxx'
DEBUG: [Common.Authentication]: User info for token DisplayId: 'xxxxxxx', Name: xxxxxxx, IdProvider: 'https://sts.windows.net/048e4758-eb3c-4d3e-
8534-630fbf0bacd5/', Uid: '06e0ff4c-6ec9-482e-b436-927c3a297f3f'
DEBUG: [Common.Authentication]: Checking token expiration, token expires '11/14/2019 19:14:06 +00:00' Comparing to '11/14/2019 18:17:24 +00:00' With thr
eshold '00:05:00', calculated time until token expiry: '00:56:41.7570328'
DEBUG: ============================ HTTP REQUEST ============================

HTTP Method:
GET

Absolute Uri:
https://management.azure.com/Subscriptions/c4fe2e04-b908-46e3-a6f5-dd5059e9f7e3/resourceGroups/cg-p-rsv-rg/providers/Microsoft.RecoveryServices/vaults/c
g-p-rstenant-rsv/backupPolicies?api-version=2017-07-01

Headers:
x-ms-client-request-id        : 090df1d2-134a-4c13-9b56-0fde4c12717a
accept-language               : en-US

Body:



DEBUG: ============================ HTTP RESPONSE ============================

Status Code:
OK

Headers:
Pragma                        : no-cache
X-Content-Type-Options        : nosniff
x-ms-request-id               : e31ea971-bf1b-4b3e-8a21-2f8025e592e1
x-ms-client-request-id        : 3d0b1362-c9af-4794-9642-eb03b756928d,3d0b1362-c9af-4794-9642-eb03b756928d
Strict-Transport-Security     : max-age=31536000; includeSubDomains
x-ms-ratelimit-remaining-subscription-reads: 11998
x-ms-correlation-request-id   : e31ea971-bf1b-4b3e-8a21-2f8025e592e1
x-ms-routing-request-id       : WESTEUROPE:20191114T181723Z:e31ea971-bf1b-4b3e-8a21-2f8025e592e1
Cache-Control                 : no-cache
Date                          : Thu, 14 Nov 2019 18:17:23 GMT
Server                        : Microsoft-IIS/10.0
X-Powered-By                  : ASP.NET

Body:
{
  "value": [
    {
      "id": "/subscriptions/c4fe2e04-b908-46e3-a6f5-dd5059e9f7e3/resourcegroups/cg-p-rsv-rg/providers/microsoft.recoveryservices/vaults/cg-p-rstenant-rs
v/backupPolicies/SqlHemera",
      "name": "SqlHemera",
      "type": "Microsoft.RecoveryServices/vaults/backupPolicies",
      "properties": {
        "backupManagementType": "AzureWorkload",
        "workLoadType": "SQLDataBase",
        "settings": {
          "timeZone": "W. Europe Standard Time",
          "issqlcompression": true,
          "isCompression": true
        },
        "subProtectionPolicy": [
          {
            "policyType": "Full",
            "schedulePolicy": {
              "schedulePolicyType": "SimpleSchedulePolicy",
              "scheduleRunFrequency": "Daily",
              "scheduleRunTimes": [
                "2019-11-09T20:00:00Z"
              ],
              "scheduleWeeklyFrequency": 0
            },
            "retentionPolicy": {
              "retentionPolicyType": "LongTermRetentionPolicy",
              "dailySchedule": {
                "retentionTimes": [
                  "2019-11-09T20:00:00Z"
                ],
                "retentionDuration": {
                  "count": 7,
                  "durationType": "Days"
                }
              },
              "weeklySchedule": {
                "daysOfTheWeek": [
                  "Sunday"
                ],
                "retentionTimes": [
                  "2019-11-09T20:00:00Z"
                ],
                "retentionDuration": {
                  "count": 5,
                  "durationType": "Weeks"
                }
              },
              "monthlySchedule": {
                "retentionScheduleFormatType": "Daily",
                "retentionScheduleDaily": {
                  "daysOfTheMonth": [
                    {
                      "date": 1,
                      "isLast": false
                    }
                  ]
                },
                "retentionTimes": [
                  "2019-11-09T20:00:00Z"
                ],
                "retentionDuration": {
                  "count": 12,
                  "durationType": "Months"
                }
              },
              "yearlySchedule": {
                "retentionScheduleFormatType": "Daily",
                "monthsOfYear": [
                  "December"
                ],
                "retentionScheduleDaily": {
                  "daysOfTheMonth": [
                    {
                      "date": 0,
                      "isLast": true
                    }
                  ]
                },
                "retentionTimes": [
                  "2019-11-09T20:00:00Z"
                ],
                "retentionDuration": {
                  "count": 7,
                  "durationType": "Years"
                }
              }
            }
          },
          {
            "policyType": "Log",
            "schedulePolicy": {
              "schedulePolicyType": "LogSchedulePolicy",
              "scheduleFrequencyInMins": 15
            },
            "retentionPolicy": {
              "retentionPolicyType": "SimpleRetentionPolicy",
              "retentionDuration": {
                "count": 7,
                "durationType": "Days"
              }
            }
          }
        ],
        "protectedItemsCount": 0
      }
    },
    {
      "id": "/subscriptions/c4fe2e04-b908-46e3-a6f5-dd5059e9f7e3/resourcegroups/cg-p-rsv-rg/providers/microsoft.recoveryservices/vaults/cg-p-rstenant-rs
v/backupPolicies/HourlyLogBackup",
      "name": "HourlyLogBackup",
      "type": "Microsoft.RecoveryServices/vaults/backupPolicies",
      "properties": {
        "backupManagementType": "AzureWorkload",
        "workLoadType": "SQLDataBase",
        "settings": {
          "timeZone": "UTC",
          "issqlcompression": false,
          "isCompression": false
        },
        "subProtectionPolicy": [
          {
            "policyType": "Full",
            "schedulePolicy": {
              "schedulePolicyType": "SimpleSchedulePolicy",
              "scheduleRunFrequency": "Daily",
              "scheduleRunTimes": [
                "2019-07-15T19:00:00Z"
              ],
              "scheduleWeeklyFrequency": 0
            },
            "retentionPolicy": {
              "retentionPolicyType": "LongTermRetentionPolicy",
              "dailySchedule": {
                "retentionTimes": [
                  "2019-07-15T19:00:00Z"
                ],
                "retentionDuration": {
                  "count": 30,
                  "durationType": "Days"
                }
              }
            }
          },
          {
            "policyType": "Log",
            "schedulePolicy": {
              "schedulePolicyType": "LogSchedulePolicy",
              "scheduleFrequencyInMins": 60
            },
            "retentionPolicy": {
              "retentionPolicyType": "SimpleRetentionPolicy",
              "retentionDuration": {
                "count": 30,
                "durationType": "Days"
              }
            }
          }
        ],
        "protectedItemsCount": 0
      }
    },
    {
      "id": "/subscriptions/c4fe2e04-b908-46e3-a6f5-dd5059e9f7e3/resourcegroups/cg-p-rsv-rg/providers/microsoft.recoveryservices/vaults/cg-p-rstenant-rs
v/backupPolicies/DefaultPolicy",
      "name": "DefaultPolicy",
      "type": "Microsoft.RecoveryServices/vaults/backupPolicies",
      "properties": {
        "backupManagementType": "AzureIaasVM",
        "instantRPDetails": {},
        "schedulePolicy": {
          "schedulePolicyType": "SimpleSchedulePolicy",
          "scheduleRunFrequency": "Daily",
          "scheduleRunTimes": [
            "2019-07-15T19:00:00Z"
          ],
          "scheduleWeeklyFrequency": 0
        },
        "retentionPolicy": {
          "retentionPolicyType": "LongTermRetentionPolicy",
          "dailySchedule": {
            "retentionTimes": [
              "2019-07-15T19:00:00Z"
            ],
            "retentionDuration": {
              "count": 30,
              "durationType": "Days"
            }
          }
        },
        "instantRpRetentionRangeInDays": 2,
        "timeZone": "UTC",
        "protectedItemsCount": 0
      }
    },
    {
      "id": "/subscriptions/c4fe2e04-b908-46e3-a6f5-dd5059e9f7e3/resourcegroups/cg-p-rsv-rg/providers/microsoft.recoveryservices/vaults/cg-p-rstenant-rs
v/backupPolicies/SqlDefault",
      "name": "SqlDefault",
      "type": "Microsoft.RecoveryServices/vaults/backupPolicies",
      "properties": {
        "backupManagementType": "AzureWorkload",
        "workLoadType": "SQLDataBase",
        "settings": {
          "timeZone": "W. Europe Standard Time",
          "issqlcompression": true,
          "isCompression": true
        },
        "subProtectionPolicy": [
          {
            "policyType": "Full",
            "schedulePolicy": {
              "schedulePolicyType": "SimpleSchedulePolicy",
              "scheduleRunFrequency": "Daily",
              "scheduleRunTimes": [
                "2019-09-10T20:00:00Z"
              ],
              "scheduleWeeklyFrequency": 0
            },
            "retentionPolicy": {
              "retentionPolicyType": "LongTermRetentionPolicy",
              "dailySchedule": {
                "retentionTimes": [
                  "2019-09-10T20:00:00Z"
                ],
                "retentionDuration": {
                  "count": 7,
                  "durationType": "Days"
                }
              },
              "weeklySchedule": {
                "daysOfTheWeek": [
                  "Sunday"
                ],
                "retentionTimes": [
                  "2019-09-10T20:00:00Z"
                ],
                "retentionDuration": {
                  "count": 5,
                  "durationType": "Weeks"
                }
              },
              "monthlySchedule": {
                "retentionScheduleFormatType": "Daily",
                "retentionScheduleDaily": {
                  "daysOfTheMonth": [
                    {
                      "date": 1,
                      "isLast": false
                    }
                  ]
                },
                "retentionTimes": [
                  "2019-09-10T20:00:00Z"
                ],
                "retentionDuration": {
                  "count": 12,
                  "durationType": "Months"
                }
              },
              "yearlySchedule": {
                "retentionScheduleFormatType": "Daily",
                "monthsOfYear": [
                  "December"
                ],
                "retentionScheduleDaily": {
                  "daysOfTheMonth": [
                    {
                      "date": 0,
                      "isLast": true
                    }
                  ]
                },
                "retentionTimes": [
                  "2019-09-10T20:00:00Z"
                ],
                "retentionDuration": {
                  "count": 7,
                  "durationType": "Years"
                }
              }
            }
          },
          {
            "policyType": "Log",
            "schedulePolicy": {
              "schedulePolicyType": "LogSchedulePolicy",
              "scheduleFrequencyInMins": 120
            },
            "retentionPolicy": {
              "retentionPolicyType": "SimpleRetentionPolicy",
              "retentionDuration": {
                "count": 7,
                "durationType": "Days"
              }
            }
          }
        ],
        "protectedItemsCount": 0
      }
    },
    {
      "id": "/subscriptions/c4fe2e04-b908-46e3-a6f5-dd5059e9f7e3/resourcegroups/cg-p-rsv-rg/providers/microsoft.recoveryservices/vaults/cg-p-rstenant-rs
v/backupPolicies/cg-tenant",
      "name": "cg-tenant",
      "type": "Microsoft.RecoveryServices/vaults/backupPolicies",
      "properties": {
        "backupManagementType": "AzureIaasVM",
        "instantRPDetails": {},
        "schedulePolicy": {
          "schedulePolicyType": "SimpleSchedulePolicy",
          "scheduleRunFrequency": "Daily",
          "scheduleRunTimes": [
            "2019-09-10T21:00:00Z"
          ],
          "scheduleWeeklyFrequency": 0
        },
        "retentionPolicy": {
          "retentionPolicyType": "LongTermRetentionPolicy",
          "dailySchedule": {
            "retentionTimes": [
              "2019-09-10T21:00:00Z"
            ],
            "retentionDuration": {
              "count": 7,
              "durationType": "Days"
            }
          },
          "weeklySchedule": {
            "daysOfTheWeek": [
              "Friday"
            ],
            "retentionTimes": [
              "2019-09-10T21:00:00Z"
            ],
            "retentionDuration": {
              "count": 5,
              "durationType": "Weeks"
            }
          },
          "monthlySchedule": {
            "retentionScheduleFormatType": "Daily",
            "retentionScheduleDaily": {
              "daysOfTheMonth": [
                {
                  "date": 1,
                  "isLast": false
                }
              ]
            },
            "retentionTimes": [
              "2019-09-10T21:00:00Z"
            ],
            "retentionDuration": {
              "count": 12,
              "durationType": "Months"
            }
          },
          "yearlySchedule": {
            "retentionScheduleFormatType": "Daily",
            "monthsOfYear": [
              "December"
            ],
            "retentionScheduleDaily": {
              "daysOfTheMonth": [
                {
                  "date": 0,
                  "isLast": true
                }
              ]
            },
            "retentionTimes": [
              "2019-09-10T21:00:00Z"
            ],
            "retentionDuration": {
              "count": 7,
              "durationType": "Years"
            }
          }
        },
        "instantRpRetentionRangeInDays": 2,
        "timeZone": "W. Europe Standard Time",
        "protectedItemsCount": 0
      }
    }
  ]
}


DEBUG: Successfully got response from service
DEBUG: Total policies in list: 5

Name                 WorkloadType       BackupManagementType BackupTime                Frequency                                IsDifferentialBackup Is
                                                                                                                                Enabled              Lo
                                                                                                                                                     gB
                                                                                                                                                     ac
                                                                                                                                                     ku
                                                                                                                                                     pE
                                                                                                                                                     na
                                                                                                                                                     bl
                                                                                                                                                     ed
----                 ------------       -------------------- ----------                ---------                                -------------------- --
SqlHemera            MSSQL              AzureWorkload        11/9/2019 7:00:00 PM      Daily                                    False                Tr
SqlDefault           MSSQL              AzureWorkload        9/10/2019 6:00:00 PM      Daily                                    False                Tr
DEBUG: AzureQoSEvent: CommandName - Get-AzRecoveryServicesBackupProtectionPolicy; IsSuccess - True; Duration - 00:00:00.7507105; Exception - ;
DEBUG: Finish sending metric.
DEBUG: 7:17:25 PM - GetAzureRmRecoveryServicesBackupProtectionPolicy end processing.
DEBUG: 7:17:25 PM - GetAzureRmRecoveryServicesBackupProtectionPolicy end processing.

```

**Comment by pvrk** [comment]
@siddharth7 : Can you please check this?

**Comment by siddharth7** [comment]
@peppekerstens I believe it has to do something with daylight saving time settings as the dates being shown in backup time column are on 2 different sides of 27th October

**Comment by peppekerstens** [comment]
@siddharth7 that is/was my guess too. You are correct when assuming these two items have been made before and after daylight saving time.  

Do I assume correct that PS eventually makes the same API calls as the GUI does? If yes, then PS should provide same result, which it does not :(

**Comment by pvrk** [comment]
@peppekerstens : Sincere apologies for the delay here. In retrospect, I am not sure I understand the issue. I understand that we have 2 policies created for the same time but 1 before the daylight savings and 1 after. The portal displays the local time using the browser and PS always shows the UTC time.  They are different in this aspect. So, can you please let us know what is the behavior you see, where and what do you want to be fixed?

**Comment by pvrk** [comment]
@peppekerstens : Another gentle reminder.

**Comment by peppekerstens** [comment]
@prvk sorry, long running issue; forgot all about it.

Time has passed, code has changed. So I re-checked the issue. now on PS7.0.3/AZ 4.5.0 and the issue remains.

If you read the above debug logs carefully, the underlaying REST API call actually provides the PS command with following time information for each policy (focus on timezone & scheduleRunTimes settings) :

      "name": "SqlHemera",
      "type": "Microsoft.RecoveryServices/vaults/backupPolicies",
      "properties": {
        "backupManagementType": "AzureWorkload",
        "workLoadType": "SQLDataBase",
        "settings": {
          "timeZone": "W. Europe Standard Time",
          "issqlcompression": true,
          "isCompression": true
        },
        "subProtectionPolicy": [
          {
            "policyType": "Full",
            "schedulePolicy": {
              "schedulePolicyType": "SimpleSchedulePolicy",
              "scheduleRunFrequency": "Daily",
              "scheduleRunTimes": [
                "2019-11-09T20:00:00Z"
              ],
              "scheduleWeeklyFrequency": 0
            },

      "name": "SqlDefault",
      "type": "Microsoft.RecoveryServices/vaults/backupPolicies",
      "properties": {
        "backupManagementType": "AzureWorkload",
        "workLoadType": "SQLDataBase",
        "settings": {
          "timeZone": "W. Europe Standard Time",
          "issqlcompression": true,
          "isCompression": true
        },
        "subProtectionPolicy": [
          {
            "policyType": "Full",
            "schedulePolicy": {
              "schedulePolicyType": "SimpleSchedulePolicy",
              "scheduleRunFrequency": "Daily",
              "scheduleRunTimes": [
                "2019-09-10T20:00:00Z"
              ],
              "scheduleWeeklyFrequency": 0
            },



The command result however is:
```powershell
$result =Get-AzRecoveryServicesBackupProtectionPolicy -VaultId $rsVault.ID | Where Name -Like "*Sql*"
$result

Name                 WorkloadType       BackupManagementType BackupTime                Frequency
----                 ------------       -------------------- ----------                ---------
SqlHemera            MSSQL              AzureWorkload        11/9/2019 7:00:00 PM      Weekly
SqlDefault           MSSQL              AzureWorkload        9/10/2019 6:00:00 PM      Daily
```

So currently is misinterprets the timezone completally (my laptop is CEST+1 right now)

It is not a PS version issue. On PowerShell 5, same result:

```powershell
:\WINDOWS\system32> $rsVault = Get-AzRecoveryServicesVault -Name 'cg-p-rstenant-rsv'
C:\WINDOWS\system32>  Get-AzRecoveryServicesBackupProtectionPolicy -VaultId $rsVault.ID | Where Name -Like "*Sql*"

Name                 WorkloadType       BackupManagementType BackupTime                Frequency
----                 ------------       -------------------- ----------                ---------
SqlHemera            MSSQL              AzureWorkload        11/9/2019 7:00:00 PM      Weekly
SqlDefault           MSSQL              AzureWorkload        9/10/2019 6:00:00 PM      Daily
```

**Comment by peppekerstens** [comment]
to be complete; interface pics:
![image](https://user-images.githubusercontent.com/13161629/91154600-fdd6a280-e6c1-11ea-84b1-690c53e54952.png)

![image](https://user-images.githubusercontent.com/13161629/91154640-08913780-e6c2-11ea-86a2-20501fde2c89.png)

**Comment by pvrk** [comment]
@peppekerstens : Thanks for the update. Will look into it and let you know.

**Comment by pvrk** [comment]
@peppekerstens : Want to clarify that portal always displays results in the local, browser time-zone (which is why both times are shown in WestEurope standard time since you are logged in the same time-zone as well) whereas PS displays time in UTC and hence we have 07:00 PM and 06:00 PM. We can make it more explicit that PS only shows UTC by modifying the table output.

Does this make sense? Please correct me if my understanding of relevant time-zones is incorrect.

**Comment by Sharmistha-Rai** [comment]
This is an Azure Backup query. I see that @pvrk is already engaged on this thread. Please assign it a tag corresponding to Backup Team.

**Comment by peppekerstens** [comment]
@pvrk looked into it once more. I am beginning to see where this all went sour. 

The backup times where set via the portal GUI.
The Recovery Services Vault/Backup Policy provides the option to select your preferred timezone (so no matter what your browser settings are). 
 
![issue10546backuppolicy](https://user-images.githubusercontent.com/13161629/100740522-25cc5500-33d9-11eb-88be-894d25b08407.png)

From this point on, backups being made by that Backup Policy 'are local time correct'.  So, when set at 20:00 local time, they are 'run' within Azure at such a time that this corresponds to '20:00 local time'.

However, when entering the PowerShell commands, this 'interface translation' is not being done and the UTC times are being displayed, even calculating in the summer/winter time displacement.

After re-iterating through documentation 
1) it is not clear that Get-AzRecoveryServicesBackupProtectionPolicy shows the UTC time format (https://docs.microsoft.com/en-us/powershell/module/az.recoveryservices/get-azrecoveryservicesbackupprotectionpolicy?view=azps-5.1.0#outputs)
2) only after reading https://docs.microsoft.com/en-us/powershell/module/az.recoveryservices/set-azrecoveryservicesbackupprotectionpolicy?view=azps-5.1.0#example-1--modify-a-backup-protection-policy I deduced the UTC time input/ouput preference

What baffles me as a end-user/operator, is that PS and Azure portal do not produce consistent output/input. I would have expected 'same experience'. The way it is done/shown now does not help.

So, after a long time, maybe this issue should become a proposal :)

Get-AzRecoveryServicesBackupProtectionPolicy could be enriched by producing both 'UTC time', 'TimeZone time' and setted 'time zone'. The last two should be easy, as they are already present within the REST output, 

Also

Based on https://docs.microsoft.com/en-us/powershell/module/az.recoveryservices/set-azrecoveryservicesbackupprotectionpolicy?view=azps-5.1.0#example-1--modify-a-backup-protection-policy the
[Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.Models.SimpleSchedulePolicy] object does not seem to have an timezone property, while there is one in the REST API. 

So on a broader scope for the RecoveryServices commands/objects; please consider to enrich them with the timezone property and 'time zone corrected time' for more consistent end-user experience/interface

**Comment by pvrk** [comment]
@peppekerstens : Good to know that we have a clarity on the final ask. I will approach this in 2 steps.

- Clarify that PS/CLI take input in UTC as opposed to Portal which takes local
- PS/CLI should also start to accept time in local/desired time-zone properties.

**Comment by pvrk** [comment]
@mepand : A request for new feature. Let's add this to our backlog.

**Comment by mepand** [comment]
@peppekerstens We have created a backlog item for providing a consistent experience wrt the timezone property and would target to improve it in H1 CY 2021. To provide more clarity on Get-AzRecoveryServicesBackupProtectionPolicy showing the UTC time format ,we have updated the column header in the output so that it is explicitly stated that the timezone reported is in UTC.  Closing the issue . Will track the feature backlog item internally.

**Comment by peppekerstens** [comment]
@pvrk and @mepand : That would be helpful I think. Thank you for your time and effort.

**Comment by peppekerstens** [comment]
@pvrk and @mepand : one last thing to note: current diplay of UTC time, triggers on the creation time of the schedule. IN my example is:

          "scheduleRunTimes": [
            "2019-11-09T20:00:00Z"
          ],

and

          "scheduleRunTimes": [
            "2019-09-10T20:00:00Z"
          ],

which is in different daylight saving times. This results in display (UTC) output to be different. 

A simple fix would/could be to check current system (UTC) time and calculate from that to display the output. This way, there is no direct need to change to much code and still provide a more consistent output compared with Azure GUI.

So
current (code) behavior seems to be:
- check time from API call
- display schedule creation times which fall out of scope of daylight saving as just UTC
- display schedule creation times which fall in of scope of daylight saving as UTC + daylight savings

Preferred behavior would be (as is Azure GUI):
- check time from API call
- check system time
- if system time is daylight savings; display all UTC time with daylight savings
- if system time is not UTC + daylight savings; display all with UTC

**Comment by mepand** [comment]
@peppekerstens Thanks for the detailed analysis, insights and feedback. We would definitely consider the points you mentioned above to ensure the behaviour is consistent with Azure GUI.
